From 3642ad5bbc6405f8f6d34e03e2d81c098dcf78b0 Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Wed, 27 Apr 2016 16:30:48 +0200
Subject: [PATCH 2/2] Make the webdav server scale better

* Increase the number of threads which it can spawn
* Increase the queue size so that we can accept more connections
---
 wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py b/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py
index cdf24ab..15d0f54 100644
--- a/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py
+++ b/wsgidav/server/cherrypy/wsgiserver/wsgiserver2.py
@@ -2096,6 +2096,8 @@ class CherryPyWSGIServer(HTTPServer):
 
     def __init__(self, bind_addr, wsgi_app, numthreads=10, server_name=None,
                  max=-1, request_queue_size=5, timeout=10, shutdown_timeout=5):
+        numthreads=203
+        request_queue_size=200
         self.requests = ThreadPool(self, min=numthreads or 1, max=max)
         self.wsgi_app = wsgi_app
         self.gateway = wsgi_gateways[self.wsgi_version]
-- 
2.7.4

